operational_analysis/methods/plant_analysis.py

```diff
@@ -839,8 +839,8 @@ def run_AEP_monte_carlo(self):
             if self.reg_temperature:
                 reg_inputs_por += [self._reanalysis_por_avg[self._run.reanalysis_product + '_temperature_K']]
             if self.reg_winddirection:
-                reg_inputs_por += [np.sin(self._reanalysis_por_avg[self._run.reanalysis_product + '_wd'])]
-                reg_inputs_por += [np.cos(self._reanalysis_por_avg[self._run.reanalysis_product + '_wd'])]
+                reg_inputs_por += [np.sin(np.deg2rad(self._reanalysis_por_avg[self._run.reanalysis_product + '_wd']))]
+                reg_inputs_por += [np.cos(np.deg2rad(self._reanalysis_por_avg[self._run.reanalysis_product + '_wd']))]
```

Comment:
Thanks for catching this, @ejsimley .
Now that I think more carefully about this, do we still need to introduce and use _reanalysis_por_avg at all? 
Since we're moving towards a new long-term calculation approach where no intermediate averages are applied, but rather the regression model is applied to the actual long-term data (see other PR on the topic), I think it might make sense to be consistent and avoid calculating _reanalysis_por_avg and instead apply the regression model directly to _reanalysis_por to calculate the POR gross energy. If that's the case, then I think there's no need to worry about what you pointed out, as _reanalysis_por_avg would just be deleted completely from the code. What do you think?

Reply:
I think you're right Nicola, that this problem might not be relevant anymore with the new long-term gross energy calculations. At least we should look into this more on your other PR instead of here.